apis/accounts/index.yaml

```diff
@@ -0,0 +1,148 @@
+get:
+  tags:
+    - Accounts
+  operationId: "ListAccounts"
+  summary: "List accounts."
+  description: |
+    List all accounts known to and decrypted to this validator client
+  security:
+    - cookieAuth: []
+  parameters:
+    - name: page_size
+      in: query
+      required: false
+      description: "The maximum number of accounts to return in the response."
+      schema:
+        type: number
+    - name: page_token
+      in: query
+      required: false
+      description: "A pagination token returned from a previous call to `ListAccounts`. that indicates where this listing should continue from."
```

Comment:
I believe we generally steer clear of pagination in this repository. I'd be a fan of that here, I can't imagine any responses being prohibitively large and it saves a lot of complexity.

Reply:
If pagination is implemented, what about sorting? What happens when I access the pages and in the mean time the list changes because of internal reorg, import, deletion?